README.md

```diff
@@ -8,9 +8,11 @@ This terraform deploys an RDS instance.
 module "rds" {
   source = "github.com/byu-oit/terraform-aws-rds?ref=v0.2.0"
 
-  identifier     = "example"
-  engine         = "mysql"
-  engine_version = "8.0"
+  identifier              = "example"
+  engine                  = "mysql"
+  engine_version          = "8.0"
+  security_group_ids      = [module.acs.rds_security_group.id]
+  cloudwatch_logs_exports = ["error", "general"]
```

Comment:
(nit) I usually don't include the optional parameters here in the usage section, but I'm fine either way

Reply:
You mean for cloudwatch_logs_exports? I put that in because the default didn't work with MySQL 8.0.11. Terraform just said that 'audit' logs don't work with MySQL 8.0.11.

Reply:
Interesting. We'll have to look into this, but for now this should be fine.